pages/common/nc.md

```diff
@@ -16,11 +16,11 @@
 
 - Serve a file:
 
-`nc -l {{port}} < {{somefile.txt}}`
+`nc -l {{port}} < {{some_file}}`
```

Comment:
Sorry for nitpicking, but isn't the "some" here redundant? Especially considering the other pages, which use a bare "file" or "filename".